kubernetes_practice/resources/setup.sh

```diff
@@ -0,0 +1,29 @@
+# pull ubuntu image
+docker pull ubuntu
+
+# use Dockerfile to build an docker image
+docker build Dockerfile
```

Comment:
Add the `-t` flag to specify a tag, aka., name for this docker image.

Refer to the manpage of the `docker` command for more information.